0.4.1: test_unedited_template fails

Hi,

when building on Debian (unstable; x86_64), I get the following:
```
___________________________ test_unedited_template ____________________________

    def test_unedited_template():
        # Test that the config file is written at most once
        config_dir = os.path.join(os.path.dirname(__file__), '..', '..')
        configuration.update_default_config('astropy', config_dir)
>       assert configuration.update_default_config('astropy', config_dir) is False
E       assert None is False
E        +  where None = <function update_default_config at 0x7f84e56d10c8>('astropy', '/tmp/astropy-test-PVRjx7/lib.linux-x86_64-2.7/astropy/config/tests/../..')
E        +    where <function update_default_config at 0x7f84e56d10c8> = configuration.update_default_config

astropy/config/tests/test_configs.py:334: AssertionError
```
Python version is 2.7

--- a/astropy/config/configuration.py
+++ b/astropy/config/configuration.py
@@ -812,7 +812,7 @@ def update_default_config(pkg, default_cfg_dir_or_fn, version=None):
 
     Raises
     ------
-    ConfigurationDefaultMissingError
+    AttributeError
         If the version number of the package could not determined.
 
     """
@@ -842,9 +842,6 @@ def update_default_config(pkg, default_cfg_dir_or_fn, version=None):
 
     if version is None:
         mod = __import__(pkg)
-        if not hasattr(mod, '__version__'):
-            raise ConfigurationDefaultMissingError(
-                'Could not determine version of package {0}'.format(pkg))
         version = mod.__version__
 
     # Don't install template files for dev versions, or we'll end up

--- a/astropy/config/configuration.py
+++ b/astropy/config/configuration.py
@@ -813,7 +813,7 @@ def update_default_config(pkg, default_cfg_dir_or_fn, version=None):
     Raises
     ------
     ConfigurationDefaultMissingError
-        If the default configuration could not be found.
+        If the version number of the package could not determined.
 
     """
 
@@ -826,7 +826,7 @@ def update_default_config(pkg, default_cfg_dir_or_fn, version=None):
         # There is no template configuration file, which basically
         # means the affiliated package is not using the configuration
         # system, so just return.
-        return
+        return False
 
     cfgfn = get_config(pkg).filename
 

